src/main/java/mate/academy/jvbookstore/validation/FieldMatchValidator.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.jvbookstore.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import java.lang.reflect.Field;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch, Object> {
+    @Override
+    public boolean isValid(Object value, ConstraintValidatorContext context) {
+        Class<?> clazz = value.getClass();
+        Field[] fields = clazz.getDeclaredFields();
+        Field.setAccessible(fields, true);
+        boolean isFirstValueSet = false;
+        Object objToMatch = null;
```

Comment:
```suggestion
        Object firstValue = null;
```
isFirstValueSet -> firstValue 
isObjToMatchSet -> objToMatch 

Think about it